docs/admin/hardware.md

```diff
@@ -34,10 +34,17 @@ validated on a regular basis.
 
 The DAOS data plane relies on [OFI libfabrics](https://ofiwg.github.io/libfabric/)
 and supports OFI
-providers for Ethernet/sockets and InfiniBand/verbs. An RDMA-capable
-fabric is preferred for better performance. DAOS can support multiple
-rails by binding different instances of the DAOS server to individual
+providers for Ethernet/tcp and InfiniBand/verbs. An RDMA-capable
+fabric is preferred for better performance.
+
+DAOS supports multiple network interfaces on the servers
+by binding different instances of the DAOS engine to individual
 network cards.
+DAOS can support multiple network interfaces on the clients,
```

Comment:
i believe that the daos_agent should normally support this transparently, without any specific settings by the user. and since this is the general hardware section, i didn't want to go into specifics on non-default agent setup here...